features/step_definitions/new_objects_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given("the following articles exists") do |table|
+    table.hashes.each do |article|
+        Article.create!(article)
```

Comment:
Please use FactoryBot when you create objects in test env. That is the practice we prefer.

Reply:
And please avoid using bang methods (in this case `create!`) unless you really know what you are doing ;-)

Reply:
Okay noted and fixed.

Reply:
In one place yes. See another comment.